cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -255,10 +257,114 @@ func (m *awsWrapper) getInstanceTypeByLaunchTemplate(launchTemplate *launchTempl
 	return aws.StringValue(instanceType), nil
 }
 
+func (m *awsWrapper) getInstanceTypesFromInstanceRequirements(policy *mixedInstancesPolicy) ([]string, error) {
+	requirementsRequest := ec2.InstanceRequirementsRequest{
+		// AcceleratorCount: &ec2.AcceleratorCountRequest{
```

Comment:
These remain to be fleshed out and supported with nil-checks. However, I wanted to first ask the question of if this can be done any simpler. Some values, such as `CpuManufacturers`, are of the same type between `ec2.InstanceRequirementsRequest` and `autoscaling.InstanceRequirementsRequest` (provided by MixedInstancesPolicy). However, type conversion doesn't cover cases like `AcceleratorCount`. Is there a cleaner way than manually building out the request object?